integreat_cms/cms/models/contact/contact.py

```diff
@@ -0,0 +1,65 @@
+from django.db import models
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+from ..abstract_base_model import AbstractBaseModel
+from ..pois.poi import POI
+from ..regions.region import Region
+
+
+class Contact(AbstractBaseModel):
+    """
+    Data model representing a contact
+    """
+
+    title = models.CharField(max_length=200, verbose_name=_("title"))
+    name = models.CharField(max_length=200, verbose_name=_("name"))
+    poi = models.ForeignKey(POI, on_delete=models.CASCADE, verbose_name=_("POI"))
```

Comment:
```suggestion
    poi = models.ForeignKey(POI, on_delete=models.PROTECT, verbose_name=_("POI"))
```
I would make this protected from deletion for safety 🦺

Reply:
> I would make this protected from deletion for safety 🦺
It would be very painful if a contact which had been embedded everywhere were deleted😭 and we should not expect users are well aware of this denger 😅

Good point! It's adjusted now, but I need a bit more time to fix the test